pkg/output/table.go

```diff
@@ -155,23 +155,37 @@ func (t *Table) printCore(writer io.Writer, auto bool) error {
 
 	w := tablewriter.NewWriter(writer)
 	w.SetAutoWrapText(auto)
+	w.SetAlignment(tablewriter.ALIGN_LEFT)
 
 	if t.isList {
+		w.SetAutoFormatHeaders(false)
+		w.SetBorder(false)
+
+		n := reflect.TypeOf(t.objects[0]).Elem().NumField()
+
 		var header []string
-		for i := 0; i < reflect.TypeOf(t.objects[0]).Elem().NumField(); i++ {
-			tag := strings.Split(reflect.TypeOf(t.objects[0]).Elem().Field(i).Tag.Get(t.format.String()), ",")
+		var alignment []int
+		for i := 0; i < n; i++ {
+			field := reflect.TypeOf(t.objects[0]).Elem().Field(i)
```

Comment:
will this cause a npe? `objects[0]`

Reply:
No we check the size of the slice above!